README.md

```diff
@@ -13,33 +13,23 @@ remote Opam repository:
 
 ## Layout of This Repository
 
-Read this to understand how to make changes
-
-    +-- compilers                       part of Opam repo (empty)
-    +-- LICENSE
-    +-- packages                        part of Opam repo
-    |   +-- upstream                    all upstream packages
-    |   |   +-- angstrom.0.1.1
-    |   |   |   +-- descr
-    |   |   |   +-- findlib
-    |   |   |   +-- opam
-    |   |   |   \-- url
-    |   |   \-- zed.1.4
-    |   |       +-- descr
-    |   |       +-- findlib
-    |   |       +-- opam
-    |   |       \-- url
-    |   \-- xs                          all XenServer packages
-    |       +-- message-switch.jonludlam#ppx
-    |       |   +-- descr
-    |       |   +-- opam
-    |       |   \-- url
-    |       \-- xenstore_transport.0.9.4
-    |           +-- descr
-    |           +-- findlib
-    |           +-- opam
-    |           \-- url
-    +-- README.md
+Packages are organised into namespaces for organisational aspects. They
+are equivalent from the perspective of the Opam user.
+
+* `upstream`: packages that we don't control with fixed versions.
+* `xs`: packages that we control, with fixed versions.
+* `xs-extra`: packages that we control, following their respective
+  master branch.
+
+    ./packages
```

Comment:
Looking at https://github.com/lindig/xs-opam/, this is not recognized as a code block sadly.